jupyter_desktop/__init__.py

```diff
@@ -24,9 +40,9 @@ def setup_desktop():
             '-SecurityTypes', 'None',
             '-rfbunixpath', sockets_path,
             '-fg',
-            ':1',
+            ':' + str(rfbport-5900),
```

Comment:
Can you add a comment here on why this is needed?

Reply:
We're using a unix socket, so the port should only be used for incoming websockify connections. I think this means this should always be `:<number of vncservers running>` i.e. `:1`?

Reply:
@manics On a multiuser system the websockify shouldn't always be the same 59xx, but is the display always :1 regardless? I haven't thought this through. :/

Reply:
Good point, I've just tried it (`docker exec -it -u root ...`, create a new user, run
`/opt/conda/bin/websockify -v --web /opt/noVNC-1.1.0 5902 --unix-target /tmp/vnc.socket2 -- vncserver -verbose -xstartup xfce4-session -SecurityTypes None -rfbunixpath /tmp/vnc.socket2 -fg :1`
have to use a different socket path, obviously).

This fails because the first server created a lock file:
```
Fatal server error:
Server is already active for display 1
        If this server is no longer running, remove /tmp/.X1-lock
        and start again.
```

However it seems to work fine if you drop the `:N` completely and let vncserver pick whatever it wants!
`/opt/conda/bin/websockify -v --web /opt/noVNC-1.1.0 5902 --unix-target /tmp/vnc.socket2 -- vncserver -verbose -xstartup xfce4-session -SecurityTypes None -rfbunixpath /tmp/vnc.socket2 -fg`
will automatically pick `:2` if there's already a server running.